app/helpers/puppetclasses_and_environments_helper.rb

```diff
@@ -28,12 +28,18 @@ def pretty_print(classes)
       if (mod = klass.gsub(/::.*/, ""))
         hash[mod] ||= []
         hash[mod] << klass
-      else
-        next
       end
     end
     hash.keys.sort.map do |key|
-      link_to key,{}, {:remote => true, :rel => "popover", :data => {"content" => hash[key].sort.join('<br>').html_safe, "original-title" => key}}
+      link_to key,{}, { :remote => true,
+                        :rel => "popover",
+                        :data => {
+                          "content" => hash[key].sort.join('<br>').html_safe,
+                          "original-title" => "#{key} has <span class='label label-info'>#{hash[key].size}</span> classes",
+                          "trigger" => "manual",
+                          "toggle" => "popover"
+                        }
+      }
```

Comment:
Closing brace should be aligned with opening brace. (like [here](https://github.com/ripcurld00d/foreman/blob/fix2135/app/helpers/application_helper.rb#L287))